metricbeat/mb/event.go

```diff
@@ -97,6 +98,9 @@ func AddMetricSetInfo(module, metricset string, event *Event) {
 	if event.Took > 0 {
 		info["rtt"] = event.Took / time.Microsecond
 	}
+	if event.Period > 0 {
```

Comment:
Can you also update the godocs to mention `period` and show `period` in the sample JSON.